docker/lr-pb/collect_polymerase_read_lengths.py

```diff
@@ -0,0 +1,198 @@
+import pysam
```

Comment:
I'm going to assume you did some basic testing with test data.

Reply:
Correct - this was tested a long time ago with a few different bam files.